SWLOR.Game.Server/Core/NWScript/Enum/animation.cs

```diff
@@ -47,6 +47,56 @@ public enum Animation
         LoopingCustom20,
         Mount1,
         Dismount1,
+        LoopingCustom21,
```

Comment:
Would prefer more descriptive names for these but it's ok for now.

Reply:
Just left it the naming convention the way It was. I don't mind changing them all if that's kosher.